### PR TITLE
Track downloads

### PR DESCRIPTION
**Pull request by AydinHassan**

### CODE REVIEW

File: src/Action/Admin/Workshop/View.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            'pageDescription' => $workshop->getDisplayName(),
+            'content'         => $inner
+        ]);
+    }
+
+    private function getLast30DayInstallGraphData(array $installs) : array
+    {
+        $end    = new DateTimeImmutable();
+        $begin  = $end->sub(new DateInterval("P30D"));
+        $end    = $end->modify('+1 day');
```
Comment (mikeymike) [inline src/Action/Admin/Workshop/View.php]: any reason not to use `add` with a dateinterval here?

File: src/Repository/DoctrineORMWorkshopInstallRepository.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            's.dateTime',
+            ':from',
+            ':to'
+        );
+
+        $qb
+            ->where($where)
+            ->andWhere('s.workshop = :workshop')
+            ->setParameters([
+                                'from' => $thirtyDaysAgo,
```
Comment (mikeymike) [inline src/Repository/DoctrineORMWorkshopInstallRepository.php]: spacing seems fucked

Comment (AydinHassan) [inline src/Repository/DoctrineORMWorkshopInstallRepository.php]: yeah my phpstorm is a turd

**Comment by AydinHassan** [comment]
@mikeymike fixed - merging now.

**Pull request merged**